recipes/mysqlclient/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mysqlclient" %}
+{% set version = "1.3.12" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "2d9ec33de39f4d9c64ad7322ede0521d85829ce36a76f9dd3d6ab76a9c8648e5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - mysql-connector-c
```

Comment:
Might need to pin, depending upon: https://github.com/lvc/upstream-tracker/issues/42